src/ansible/roles/packages/tasks/CentOS8.yml

```diff
@@ -30,7 +30,7 @@
   - name: Enable IdM module
     shell: |
       dnf module enable -y idm:DL1
-  when: "'base_ground' in group_names"
+  when: "'base_ground' in group_names or 'base' in group_names"
```

Comment:
This won't work for containers as the step will be repeated every time a new layer is added.

Reply:
Ok, I probably misundestood that when we wer btalking about that.
I will fix it.

Reply:
Fixed.